### PR TITLE
README's V 1.0

### LINKED ISSUE #114: Flight Documentation Update
# Flight Documentation Update

## Problem

Flight documentation should be updated to reflect most recent state of flight code.

## Solution

Each directory under the `flight/` folder should contain a `README.md` with a high-level overview of what each directory contains and how modules in it are used/interacted with from other parts of the code.

### PR DESCRIPTION
**Pull request by Lmyszka4**
An exquisite collection of README's designed to educate the reader as to how each of the functions collaborate to ensure the success of the mission.

### CODE REVIEW

File: flight/README.md [hunk 1]
```diff
@@ -1 +1,11 @@
 # Flight
+
+This is the repository that holds the flight movement code. The code here allows the drone to complete all of its required functions during competition flight. Every function for connection and takeoff, to return and landing and everything else inbetween is here.
```
Comment (fallscameron01) [inline flight/README.md]: This is folder

**Review by fallscameron01 — CHANGES_REQUESTED** [review]

File: flight/README.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+Config contains the parameters the drone will abide by for the entire duration of the flight, such as takeoff altitude, max flight altitude, and waypoint wait time.
+
+Extract GPS contains the function for extracting the waypoint coordinates from a JSON file into a single dict for the SUAS competition.
+
+Flight contains the functions necessary for initializing the drone's flight and monitoring the statuses of the drone.
```
Comment (cjhr95) [inline flight/README.md]: Status, singular plz

File: flight/data/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+**Data Subdirectory**
+
+A collection of GPS coordinates used for test flights.
+This json file is used for flight data at David Hoffman's plot of land.
```
Comment (cjhr95) [inline flight/data/README.md]: Label which file you're referring to specifically with this line

File: flight/maestro/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+**Maestro Subdirectory**
+
+Maestro includes a class to communicate with the Maestro board
+It has functions for every command that it supports
```
Comment (cjhr95) [inline flight/maestro/README.md:4]: Provide some examples on what kind of commands are provided

**Review by cjhr95 — CHANGES_REQUESTED** [review]
More professional

File: flight/test_files/README.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+**Test Files Subdirectory**
+
+Contains files and coordinates for running test flights at both the golf course and David Hoffman's land.
+
+Both files have basic drone functions for creating connection with drone,
+logging takeoff status, flying to each given waypoint, and returning home.
+
+Altitude is set to 100 feet for both.
```
Comment (cjhr95) [inline flight/test_files/README.md]: This isn't entirely true, the testing file have multiple altitudes for different waypoints.

**Review by cjhr95 — CHANGES_REQUESTED** [review]
One itty-bitty little thing left

**Review by cjhr95 — APPROVED** [review]
I GUESS this will suffice

**Review by fallscameron01 — APPROVED** [review]
Very nice, looks good

**Review by fallscameron01 — APPROVED** [review]

**Pull request merged**